tests/validations/test_measures.py

```diff
@@ -177,9 +177,9 @@ def test_measure_alias_is_set_when_required() -> None:
     missing_alias_file = YamlConfigFile(filepath="inline_for_test", contents=yaml_contents)
     model = parse_yaml_files_to_validation_ready_model([missing_alias_file])
```

Comment:
Looks like we should also update this to be something like
```python
semantic_manifest = parse_yaml_files_to_validation_ready_semantic_manifest([missing_alias_file])
```
But that should probably be a separate PR which updates `parse_yaml_files_to_validation_ready_model` to `parse_yaml_files_to_validation_ready_semantic_manifest` and all the follow on changes that entails.